src/redux/Topic/saga.js

```diff
@@ -6,6 +6,40 @@ import { request } from '../../network/httpRequest';
 import { satoshiToDecimal } from '../../helpers/utility';
 import Routes from '../../network/routes';
 
+export function* getEventEscrowAmountHandler() {
+  yield takeEvery(actions.GET_EVENT_ESCROW_AMOUNT, function* getEventEscrowAmountRequest(action) {
+    try {
+      const {
+        senderAddress,
+      } = action.params;
+
+      const options = {
+        method: 'POST',
+        body: JSON.stringify({
+          senderAddress,
+        }),
+        headers: { 'Content-Type': 'application/json' },
+      };
+
+      const result = yield call(request, Routes.eventEscrowAmount, options);
+      const eventEscrowAmount = satoshiToDecimal(result[0]);
```

Comment:
why [0]? in order words, why is result an array?

Reply:
The key is actually `"0"` which corresponds to the first return param.